fdbclient/include/fdbclient/GetEncryptCipherKeys.actor.h

```diff
@@ -317,15 +318,24 @@ ACTOR template <class T>
 Future<TextAndHeaderCipherKeys> getEncryptCipherKeys(Reference<AsyncVar<T> const> db,
                                                      BlobCipherEncryptHeader header,
                                                      BlobCipherMetrics::UsageType usageType) {
-	std::unordered_set<BlobCipherDetails> cipherDetails{ header.cipherTextDetails, header.cipherHeaderDetails };
+	ASSERT(header.cipherTextDetails.isValid());
+	std::unordered_set<BlobCipherDetails> cipherDetails{ header.cipherTextDetails };
+	if (header.cipherHeaderDetails.isValid()) {
```

Comment:
Why do we need this logic if we're asserting that `header.cipherTextDetails.isValid()` in line 321 above?

Reply:
good catch. must be mistake when I merge code. thanks.